mlrun/runtimes/function.py

```
@@ -596,9 +617,12 @@ def get_nuclio_deploy_status(
     api_address = find_dashboard_url(dashboard or mlconf.nuclio_dashboard_url)
     name = get_fullname(name, project, tag)
 
-    state, address, last_log_timestamp, outputs = get_deploy_status(
-        api_address, name, last_log_timestamp, verbose
-    )
+    try:
+        state, address, last_log_timestamp, outputs = get_deploy_status(
+            api_address, name, last_log_timestamp, verbose
+        )
+    except DeployError:
+        return "", "", name, 0, ""
```

Why you're hiding failure to get the deployment status ?